example_scripts/vhi_test.py

```diff
@@ -0,0 +1,39 @@
+import os
```

Comment:
test script for working in ipython. here for posterity but DEFinitely not going to `ml_drought` repo